R/importSNV.TCGA.R

```diff
@@ -0,0 +1,268 @@
+# importSNV.TCGA.R
+
+#' Import MAF files from source and converts them to rSNV files.
+#'
+#' \code{importSNV.TCGA} Imports data from one or more files of MAF format, creates rSNV or adds to
+#' existing rSNV file, returns rSNV file.
+#'
+#' @section Validations: validations sections has 3 functions - function that checks MAF file extension,
+#' function that checks MAF file header, function that checks rSNV file header.
+#'
+#' @section Read MAF file: maf file is read by extracting require columns from maf file followed
+#' by checking if every item in the class, type and chr columns are from a known and required list of values.
+#' These columns as a data table are then added to an rSNV file.
+#'
+#' @section Write Log: log section writes file processing progress along with other errors
+#' that may occur.
+#'
+#' @param fMAF vector of MAF file names.
+#' @param rSNV rSNV file to merged new MAF data with.
+#' @param na.rm Remove all rows with missing or "NA" values and drop them from data frame.
+#' @param silent Controls whether output to console should be suppressed. FALSE
+#'   by default.
+#' @param writeLog Controls whether writing the result to the global logfile is
+#'   enabled. TRUE by default.
+#'
```

Comment:
My comment about @return did not mean you should remove it. You should specify that nothing is returned.